src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import core.basesyntax.model.Figure;
+import core.basesyntax.services.FigureSupplier;
+
+public class Application {
+    public static void main(String[] args) {
+
+        Figure[] figures = new Figure[]{
+                new Circle(), new IsoscelesTrapezoid(),
+                new Rectangle(),
+                new RightTriangle(),
+                new Square(),
+                new FigureSupplier().getDefaultFigure(),
+                new FigureSupplier().getDefaultFigure(),
+                new FigureSupplier().getDefaultFigure()
+        };
+        for (Figure figure : figures) {
```

Comment:
You should use the `FigureSupplier` to get the Figure

Reply:
The thing is, I read the requirements, but that was the best option I could think of. Today, I've got a idea, so I'm going to try that out.